src/main/java/org/codeforamerica/messaging/models/Message.java

```diff
@@ -38,4 +38,9 @@ public class Message {
     private OffsetDateTime creationTimestamp;
     @UpdateTimestamp
     private OffsetDateTime updateTimestamp;
+    @Builder.Default
+    String templateName = "default";
```

Comment:
Should this be an association with `Template`?

Reply:
Yes, I'll fix that